### ISSUE TITLE
implement postgresql foreign data wrapper client

### ISSUE THREAD

**Issue by agentm** [agentm/project-m36#158]
At the end of @diogob 's Toronto presentation, an audience member asked how one might use Project:M36 with another database. @diogob suggested that it would be possible to implement a postgresql FDW interface to Project:M36, perhaps through Project:M36's websocket server.

I'm not sure how useful such an FDW might be, but it would certainly be an interesting exploratory project.

I have considered an SQL interface to Project:M36 in the past (dis-affectionately named "SQLegacy"), but it would be a painful, self-defeating, shoe-horning effort because one of the points of this project is to point out the flaws in SQL. Therefore, running SQL queries for Project:M36 to interpret would likely be lossy- even when merely considering SQL's bag of tuples vs. Project:M36's set of tuples assumption.

**Comment by agentm** [comment]
Indeed- it's linked in our [README](https://github.com/agentm/project-m36#community) and worth watching. The audio cuts out a few times, but it returns for the important parts. The audience asked some interesting questions as well.

**Comment by theobat** [comment]
That would be a very very important feature considering -for example- the state of GIS tools in haskell (as opposed to PostGIS in postgres). Or perhaps we could adapt postgis to projectM36 ? but I doubt it, it seems very tied to the SQL language and its flaws.
